src/shared/components/driverAnnotations/TierAnnotationControls.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import { observer } from 'mobx-react';
+import autobind from 'autobind-decorator';
+import {
+    IDriverAnnotationControlsHandlers,
+    IDriverAnnotationControlsState,
+} from '../../alterationFiltering/AnnotationFilteringSettings';
+import 'rc-tooltip/assets/bootstrap_white.css';
+import _ from 'lodash';
+import { computed } from 'mobx';
+
+export interface IDriverAnnotationControlsProps {
+    state: IDriverAnnotationControlsState;
+    handlers: IDriverAnnotationControlsHandlers;
+}
+
+enum EVENT_KEY {
+    distinguishDrivers = '0',
+    customDriverBinaryAnnotation = '5',
+}
+
+@observer
+export default class TierAnnotationControls extends React.Component<
```

Comment:
more general name?

Reply:
should file be named TierAndAnnotationControls?

Reply:
Renamed the components.